environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/base/EnvironmentNetworkBase.java

```diff
@@ -68,6 +69,9 @@
     @ApiModelProperty(EnvironmentModelDescription.GCP_SPECIFIC_PARAMETERS)
     private EnvironmentNetworkGcpParams gcp;
 
+    @ApiModelProperty(EnvironmentModelDescription.LOADBALANCER_STATUS)
```

Comment:
could you please add the available values here? It will be shown in the swagger definition

Reply:
I'm not sure what you mean here. do you have an example where this is also done?

Reply:
Hi, it looks like nobody add the allowable values to the swagger, so you can ignore this.